index.js

```diff
@@ -112,6 +122,43 @@ class Conf {
 		}
 	}
 
+	_migrate(migrations = {}, versionToMigrate = '0.0.0') {
+		const MIGRATION_KEY = '__conf-migrated-version__';
+
+		const previousMigratedVersion = this.get(MIGRATION_KEY, '0.0.0');
+
+		const olderVersionsWithMigrations = Object.keys(migrations)
+			.filter(version => this._shouldVersionPerformMigration(version, previousMigratedVersion, versionToMigrate))
+			.sort(semver.compare);
+
+		const migrationsToRun = olderVersionsWithMigrations.map(version => migrations[version]);
+
+		migrationsToRun.forEach(migration => migration(this));
+
+		this.set(MIGRATION_KEY, versionToMigrate);
+	}
+
+	/**
+	 * Specifies if a version is suitable to run its migrations.
```

Comment:
@sindresorhus We didn't have any jsdocs in the remaining file, however, I found it appropriate to add internal documentation to this function.


If you prefer, I can either:

1. Delete this jsdoc
2. Add jsdocs to all the other functions.

I'm fine with either option, and I don't mind the work of last option 😄

Reply:
I am totally in favor of having jsdocs. However, as adding jsdocs to all the other functions is not in the scope of this PR, I would like to propose a different approach:

1) Keep this jsdoc and create a new PR with jsdocs to all the remaining functions.
2) Create a new PR following this one with jsdocs for everything.

I prefer the first approach, but let's hear from @sindresorhus

Reply:
@rafaelramalho19 bumping this thread to ensure you didn’t forgot to remove the jsdoc.

Reply:
I didn't, I'm just waiting on @sindresorhus answer 😄 

But I can remove it, for the time being, and add it on a later merge request.